keyboards/mechlovin/adelais/standard_led/config.h

```diff
@@ -13,3 +13,8 @@
 #define RGBLED_NUM 23
 #define RGBLIGHT_LIMIT_VAL 255
 #define RGBLIGHT_ANIMATIONS
+
+#define ENCODERS_PAD_A { A6, A4, B7 }
+#define ENCODERS_PAD_B { A5, A3, B6 }
+#define ENCODER_RESOLUTION 4
```

Comment:
Wouldn't it make more sense to move this to `keyboards/mechlovin/adelais/standard_led/rev3/config.h`?  
Therefore the feature is not enabled for everybody.

Reply:
thanks.
I just reread the tutorial about sub_folder 😄